ffi/capi/README.md

```diff
@@ -1,6 +1,19 @@
 # icu_capi [![crates.io](http://meritbadge.herokuapp.com/icu_capi)](https://crates.io/crates/icu_capi)
 
+This module contains the C FFI for ICU4X. Currently it is also used as the source
```

Comment:
I'd invert this, this contains the main source of truth for ICU4X's Diplomat-generated bindings as well as the generated C headers.

Perhaps we should be moving the `src` directory to the parent here, and having cpp, c, and wasm subfolders.

Reply:
That seems like a better organization to me.